source/home-page/assets/script.js

```diff
@@ -53,35 +53,24 @@ let images = {
  });
  */
 
-async function makeThumbnails(request){
-  let vals = await fetch(`https://api.spoonacular.com/recipes/complexSearch?apiKey=7e76c41b05734b4ab2265cebeb26cd12&maxReadyTime=${request}&addRecipeInformation=True`).then((response) => response.json());
-  for (let i = 0; i<15; i++ ){
-      let element = document.createElement("article");
-      element.id = "tile"
-      let recipe = vals.results[i]
-      console.log(recipe)
-      let title = document.createElement("h1")
-      title.id = "title"
-      title.textContent = recipe.title
-      element.appendChild(title)
-      let image = document.createElement("img")
-      image.src = recipe.image;
-      element.appendChild(image)
-      let circle = document.createElement("div")
-      circle.id = "circle"
-      circle.textContent = "<" + recipe.readyInMinutes + "min";
-      element.appendChild(circle)
-      let ElemIngredient  = document.createElement("div")
-      ElemIngredient.id = "ingredients"
-      console.log( document.getElementsByClassName("slide")[0])
-      let place = 0; 
-      if (request == '30'){
-        place = 1
-      }
-      if (request == '60'){
-        place = 2
+
+  async function makeThumbnails(request){
+    let vals = await fetch(`https://api.spoonacular.com/recipes/complexSearch?apiKey=7e76c41b05734b4ab2265cebeb26cd12&maxReadyTime=${request}&addRecipeInformation=True`).then((response) => response.json());
+    for (let i = 0; i<15; i++ ){
+      let recipeTile = document.createElement('search-recipe');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Big-Bytez/cse110-fa21-group19/pullRequest?prid=8430255)
```suggestion
      let recipeTile = document.createElement("search-recipe");
```